### PR TITLE
Optional icon for SelectInput

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
Снимка от 2019-12-10 17-34-07

### CODE REVIEW

File: packages/selectInput/components/SelectInput.tsx [hunk 1]
```diff
@@ -42,6 +42,10 @@ export interface SelectInputProps extends React.HTMLProps<HTMLSelectElement> {
    * hintContent is text or a ReactNode that is displayed directly under the input with additional information about the expected input.
    */
   hintContent?: React.ReactNode;
+  /**
+   * Optional icon to be displayed before the input value.
+   */
+  icon?: React.ReactElement<HTMLElement> | string;
```
Comment (mperrotti) [inline packages/selectInput/components/SelectInput.tsx]: Can we rename this prop to `iconStart`? That's the prop name we use for text inputs and buttons

File: packages/selectInput/style.ts [hunk 2]
```diff
@@ -39,3 +39,7 @@ export const selectIcon = css`
   top: 50%;
   transform: translateY(-50%);
 `;
+
+export const optionalIcon = css`
+  margin: auto;
```
Comment (mperrotti) [inline packages/selectInput/style.ts]: The icon is right up against the text. I recommend changing this to `margin: auto ${spaceXs} auto 0`

**Comment by mperrotti** [comment]
@GeorgiSTodorov - Looks good! I just pushed up a commit so that the `iconStart` prop can accept an icon shape ID as well as an element

**Comment by GeorgiSTodorov** [comment]
Thank you, @mperrotti .

**Pull request merged**